tests/book.test.js

```diff
@@ -0,0 +1,161 @@
+/* tests/book.test.js */
+const { expect } = require('chai');
+const request = require('supertest');
+const { Book } = require('../src/models');
+const app = require('../src/app');
+
+describe('/books', () => {
+  before(async () => Book.sequelize.sync());
+
+  beforeEach(async () => {
+    await Book.destroy({ where: {} });
+  });
+
+  describe('with no records in the database', () => {
+    describe('POST /books', () => {
+      it('creates a new book in the database', async () => {
```

Comment:
This test needs looking at, as its currently failing and looks like a work in progress